### PR TITLE
add dqn pytorch

### PR DESCRIPTION
**Pull request by Asugawara**
So sorry...
I made a mistake in https://github.com/deepmind/open_spiel/pull/443

So, I open New PR...

>In this PR, I added implements DQN in PyTorch benefitting from the TF.
The experiment using breakthrough does not show a difference between PyTorch and TF.
Let me know I should experiment with other game if you need.

if PR is approved, before import I'll delete .ipynb images.
ref:https://github.com/deepmind/open_spiel/pull/425#discussion_r522966683

### CODE REVIEW

File: open_spiel/python/pytorch/dqn_test.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
+# Copyright 2019 DeepMind Technologies Ltd. All rights reserved.
```
Comment (Asugawara) [inline open_spiel/python/pytorch/dqn_test.py:1]: I will remove this file.

File: open_spiel/python/CMakeLists.txt [hunk 2]
```diff
@@ -153,6 +153,7 @@ set(PYTHON_TESTS ${PYTHON_TESTS}
   utils/lru_cache_test.py
   utils/spawn_test.py
   pytorch/rcfr_pytorch_test.py
+  pytorch/dqn_test.py
```
Comment (lanctot) [inline open_spiel/python/CMakeLists.txt]: If you remove dqn_test.py then make sure to change this to pytorch/dqn_pytorch_test.py

Comment (Asugawara) [reply open_spiel/python/CMakeLists.txt]: Yes, I changed!

**Comment by lanctot** [comment]
Ah this last test is failing because it is too long. We will fix that on our end.

**Comment by lanctot** [comment]
Oh wait, I still don't see a commit to remove dqn_test.py and fix CMakeLists.txt. Did you forget to push?

**Comment by Asugawara** [comment]
Sorry to confuse you!
[In this commit](https://github.com/deepmind/open_spiel/pull/444/commits/411fc3b3265240b199b2a7cb5ac4581c5d87a35f)  , I rename dqn_test -> dqn_pytorch_test in CMakelist but don't remove dqn_test.py. I was going to push after travis Ci going smoothly.
Now, I push it!

**Comment by lanctot** [comment]
@Asugawara: good news and (very slightly) bad news! 

Good news is, I got the example @michalsustr suggested ready to go and added to our TF dqn test. So it'll be easy to add to yours when I import it.

(Slightly) Bad news: I can't import it, because our import script is failing on a forbidden keyword contained in the images embedded in the colab again. :) Can you remove the images? Thanks!

**Comment by Asugawara** [comment]
Thanks, Dr. @lanctot !
>Good news

Oh, I see! I'm looking forward to adding the example!
 
>(Slightly) Bad news

I was surprised by the bad news(　ﾟдﾟ) However, I was relieved that one is trivial.
I was going to remove images. Thank you for your notice!

**Comment by lanctot** [comment]
> I was surprised by the bad news(　ﾟдﾟ) However, I was relieved that one is trivial.

Hahah, well I needed some dramatic effect! 

> I was going to remove images. Thank you for your notice!

Ok it's imported and it passed the simple test (which you will see when it gets merged in Monday's update).  Thanks!

How do you feel about NFSP next? ;-) Would _love_ to have that one in PyTorch!

**Comment by Asugawara** [comment]
>Hahah, well I needed some dramatic effect!

That's a nice direction! I like it!

>Ok it's imported and it passed the simple test (which you will see when it gets merged in Monday's update). Thanks!
>How do you feel about NFSP next? ;-) Would love to have that one in PyTorch!

Sure!
Yes, I'm going to start it after importing this PR because the NFSP algorithm imports DQN!

**Pull request merged**